CONTRIBUTING.rst

```diff
@@ -288,6 +288,47 @@ Each function and class must come with a “docstring” at the top of the funct
 using numpydoc_ formatting.
 The docstring must summarize what the function does and document every parameter.
 
+If an argument takes in a default value, it should be described
+with the type definition of that argument.
+
+See the examples below:
+
+.. code-block:: python
+
+      def Good(x, y=1, z=None):
+      """Show how parameters are documented.
+
+      Parameters
+      ----------
+      x : :obj:`int`
+            X
+
+      y : :obj:`int`, default=1
```

Comment:
I thought that the recommended way was:
```suggestion
      y : :obj:`int`, optional
           Some parameter description.
           default=1
```
Did I miss something ?

Reply:
Ha. I think then we misunderstood what we had agreed on. That was why I also wanted to have a small PR like this to update the maintainers doc and to make 100% sure we were all on the same page before proceeding.